client/galaxy/scripts/apps/analysis.js

```diff
@@ -11,6 +11,7 @@ var jQuery = require( 'jquery' ),
     UserPreferences = require( 'mvc/user/user-preferences' );
     CustomBuilds = require( 'mvc/user/user-custom-builds' );
     Tours = require( 'mvc/tours' );
+    Workflows = require( 'mvc/workflow/workflow' );
```

Comment:
This is made global due to the trailing semicolons above.  Fixed in https://github.com/bgruening/galaxy/pull/35/.